### ISSUE TITLE
Wooting II coloring is broken and startup takes forever

### ISSUE THREAD

**Issue by stdedos** [antonpup/Aurora#2046]
### Actual Behaviour

Wooting II coloring is broken and startup takes forever (mouse works)
![20200526_200828-opt](https://user-images.githubusercontent.com/133706/82933370-75494880-9f92-11ea-919f-2eb657420f9f.jpg)

### Aurora Version:

0.8.0rc1 (please make that text copyable from inside Aurora)

### Previously an Issue?

No 

[2020-05-26 19.58.32.log.zip](https://github.com/antonpup/Aurora/files/4683847/2020-05-26.19.58.32.log.zip)

**Comment by diogotr7** [comment]
First of all i want to say thank you for reporting all of the issues. At the end of the day it only makes aurora a better program and i've not handled all the bugs well because i've been busy with university.

No changes were made to the wooting support. Are you sure this only started happening this update? I don't have my wooting one with me to test due to the quarantine but i genuinely can't see why it would break. I have seen some issues with rgb + wootility on the wooting discord, i'm not sure if this is related. 

We haven;t updated to the newest wooting sdk version either since i think it would be easier to do once i have the keyboard available to test with.

**Comment by stdedos** [comment]
> First of all i want to say thank you for reporting all of the issues. At the end of the day it only makes aurora a better program and i've not handled all the bugs well because i've been busy with university.

I am glad we see eye to eye on this one. Fingers crossed you'll find time 😄 

> No changes were made to the wooting support. Are you sure this only started happening this update?

Yes. Rather, it got "exaggerated".

So, the issue more descriptively is this:
* I started 0.8.0rc1
* Startup was taking forever, and all keyboard input was blocked
* I dc'ed and reconnected kb

No colouring.

And I can replicate with:
* _Aurora was restarted for testing https://github.com/antonpup/Aurora/pull/1903#issuecomment-634199109, and colouring worked (though it appears a bit more vibrant from 0.7)_
* dc and reconnect kb

In this case I don't know what used to happen, but now (a) all kb input is blocked (fixed by yet-again dc/rc) and (b) colouring is not applied (I am getting my stock/wootility config of sliding rainbow)

**Comment by diogotr7** [comment]
Do you have wootity / razer chroam wootility integration open?

Do you have a beta firmware?

Again, i wish i could test but i dont have my wooting with me :(

**Comment by stdedos** [comment]
> Do you have wootity / razer chroam wootility integration open?

No, usually I open it when I need to reset whatever Aurora does (when Aurora dies)

> Do you have a beta firmware?

No, only whatever Wootility brings (I think there was an update _sometime recently_, but I cannot help you: I don't remember when was that, neither do I remember when was the last time I used Aurora. I just heard your RC notice and updated.)

> Again, i wish i could test but i dont have my wooting with me :(

![fb-care](https://media.tenor.com/images/d16ca95bb762e1717839d0c01040b19c/tenor.gif)😕
